distsql/distsql.go

```diff
@@ -220,9 +225,24 @@ func Select(ctx context.Context, sctx sessionctx.Context, kvReq *kv.Request, fie
 		hook.(func(*kv.Request))(kvReq)
 	}
 
+	child := tracing.ChildSpanFromContxt(ctx, "distsql_select")
+	defer child.Finish()
+
 	if !sctx.GetSessionVars().EnableStreaming {
 		kvReq.Streaming = false
 	}
+
+	for _, key := range kvReq.KeyRanges {
```

Comment:
Is there a session variable to control the behavior? 
I don't like the performance degrade if the feature is not used.

Reply:
length of KeyRangs  is constant. The complexity is just `O(1)`; hence performance degradation not need to consider.

Reply:
By saying it is a constant, I mean it will not grow linearly. In general, even it is 1000 or much larger, it is still constant.